soroban-env-host/benches/common/measure.rs

```diff
@@ -208,6 +208,16 @@ pub trait HostCostMeasurement: Sized {
         Self::new_random_case(host, rng, 1)
     }
 
+    /// Baseline case is absolute minimal cost instance. It is used for estimating for the
+    /// "background noise" of the measurement. The best-case is the cheapest but valid instance.
+    /// The baseline-case may not be an actual valid case.
+    fn new_baseline_case(
```

Comment:
One thing I'm unclear on about "baselines": are they going to wind up present as a constant term in the linear model? I would think so, but I'm just checking.

Reply:
That's right, the baseline would end up being a constant term in the linear model, if not subtracted. This is a small issue in most of of the cost types but there are some with large overhead (such as setting up wasm insrns) in which this has to be taken care of (unless we take a very large number wasm insrns).